stroeer/core/v1/article.proto

```diff
@@ -961,6 +962,54 @@ message Body {
   repeated BodyNode children = 1;
 }
 
+// todo: rename to `Body`
+message BodyPart {
+  /*
+  * ### `types`
+  * | type | description |
+  * | -----| ------------|
+  * | `body` | formatted text and inlined elements |
+  * | `disclaimer` | formatted text that holds a disclaimer, e.g. used in `/gesundheit/` or `/kaufbertatung/` |
+  * | `infobox` | todo |
+  * | `table_of_contents` | todo |
+  * | `article_sources`| "Quellenapparat", list like structure containing `children[type=article_source]`. See [Samples](#article_sources). |
+  * | `article_source` | A single "Quellenapparat" item usually containing `text` or `a` |
+  */
+  string type = 1;
```

Comment:
An `enum` would be perfect. Maybe something before 1.0

Reply:
go for it. I don't see much updates here. And if something changes, it would most probably an addition. Unlikely, that stuff gets removed.

Reply:
@bobaaaaa do you create that enum?